crates/sui-indexer/README.md

```diff
@@ -22,7 +22,12 @@ diesel setup
 # and then run 
 diesel migration run
 ```
-2. cargo run under `/sui-indexer`
+2. checkout [the latest devnet release](https://github.com/MystenLabs/sui/releases), otherwise API version mismatch could cause errors
```

Comment:
```suggestion
2. check out [the latest Sui release](https://github.com/MystenLabs/sui/releases) to make sure you are using the same version of Sui that is running on Devnet
```
Is check out the best / only way to get the latest release? This is getting the release branch vs the release, right? But the release branch is cut up to a week before so one could check out the latest release branch before it is deployed to Devnet